src/home/NotificationBox.tsx

```diff
@@ -36,6 +36,7 @@ import Logger from 'src/utils/Logger'
 
 const TAG = 'NotificationBox'
 // Priority of static notifications
+const KOLEKTIVO_NOTIFICATTION_PRIORITY = 1100
```

Comment:
This is not higher priority that backup.

Reply:
fixed